### PR TITLE
Update last selected picture on exit from full screen picture view (play picture)

### PR DESCRIPTION
**Pull request by bnk1**
## Description

Update last selected picture on exit from full screen picture view (play picture):
When exiting full screen picture mode (play picture - special case of slideshow) - update the pictures list to the last viewed picture.
This doesn't affect "Slideshow" only full screen view of a picture.

## Motivation and context

This is the natural behaviour of picture viewers.

## How has this been tested?

My machine.

## What is the effect on users?

None

## Types of change

- [ ] **Bug fix** (non-breaking change which fixes an issue)
- [ ] **Clean up** (non-breaking change which removes non-working, unmaintained functionality)
- [x ] **Improvement** (non-breaking change which improves existing functionality)
- [ ] **New feature** (non-breaking change which adds functionality)
- [ ] **Breaking change** (fix or feature that will cause existing functionality to change)
- [ ] **Cosmetic change** (non-breaking change that doesn't touch code)
- [ ] **Student submission** (PR was done for educational purposes and will be treated as such)
- [ ] **None of the above** (please explain below)

## Checklist:

- [ ] My code follows the **[Code Guidelines](https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md)** of this project 
- [ ] My change requires a change to the documentation, either Doxygen or wiki
- [ ] I have updated the documentation accordingly
- [ x] I have read the **[Contributing](https://github.com/xbmc/xbmc/blob/master/docs/CONTRIBUTING.md)** document
- [ ] I have added tests to cover my change
- [ ] All new and existing tests passed

### CODE REVIEW

**Review by neo1973 — APPROVED** [review]
Thanks for the the contribution, it does what it should :+1:

I'll merge this in about a week if no one complains.

File: xbmc/pictures/GUIWindowPictures.cpp [hunk 1]
```diff
@@ -352,7 +352,8 @@ bool CGUIWindowPictures::ShowPicture(int iItem, bool startSlideShow)
   // look into using OnPlay announce!
   m_slideShowStarted = true;
   CServiceBroker::GetGUI()->GetWindowManager().ActivateWindow(WINDOW_SLIDESHOW);
-
+  if (!startSlideShow)
+    m_viewControl.SetSelectedItem(slideShow.GetCurrentSlide()->GetPath());
```
Comment (ksooo) [inline xbmc/pictures/GUIWindowPictures.cpp:356]: For better readability I would insert a blank line before the `return` statement on next line, like it was before your change.

**Comment by ksooo** [comment]
@bnk1 please squash down to a single commit - you meanwhile should know this works, right?

**Comment by neo1973** [comment]
@bnk1 Please squash the commits so I can merge this. Thank you!

**Pull request merged**